cmake/thirdparty/SetupSeracThirdParty.cmake

```diff
@@ -514,6 +533,11 @@ if (NOT SERAC_THIRD_PARTY_LIBRARIES_FOUND)
             $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/tribol/include>
             $<INSTALL_INTERFACE:include>
         )
+        target_include_directories(tribol_shared PUBLIC
+            $<BUILD_INTERFACE:${tribol_repo_dir}/src>
+            $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/tribol/include>
+            $<INSTALL_INTERFACE:include>
+        )
```

Comment:
what is the difference between tribol and tribol_shared?

Reply:
tribol_shared is for stuff that redecomp and tribol use.  There was some tribol stuff I wanted to use in redecomp, but since redecomp is a Tribol dependency, I couldn't access it without creating a new library, hence tribol_shared.